src/videoConfProvider.ts

```diff
@@ -164,13 +166,19 @@ export class BBBProvider implements IVideoConfProvider {
 		this.app.getLogger().log('Creating new meeting', call._id);
 
 		const meetingID = call._id;
-
+		const settings = this.app.getAccessors().environmentReader.getSettings();
+		const guestPolicy = await settings.getValueById(AppSetting.GuestPolicy);
+		const welcomeMsg = await settings.getValueById(AppSetting.WelcomeMsg);
+		const logoutURL = await settings.getValueById(AppSetting.LogoutURL);
+		
 		const createUrl = this.getUrlFor('create', {
 			name: call.title || 'Rocket.Chat',
 			meetingID,
 			attendeePW: 'rocket.chat.attendee',
 			moderatorPW: 'rocket.chat.moderator',
-			welcome: '<br>Welcome to <b>%%CONFNAME%%</b>!',
+			welcome: welcomeMsg,
+			guestPolicy : guestPolicy,
+			logoutURL: logoutURL,
```

Comment:
We should validate the value of the `guestPolicy` and `logoutURL` settings and ommit sending them if they are missing or invalid.